python/pip_install/private/generate_whl_library_build_bazel.bzl

```diff
@@ -86,7 +86,9 @@ py_library(
 """
 
 def generate_whl_library_build_bazel(
+        *,
```

Comment:
nit: can we call it `*args`?

I'm not entirely following why we need this? Can you elaborate?

Reply:
This is to force all of the arguments to be specified as keyword arguments. this function has lots of arguments and supporting positional arguments may be a little bit error prone.

Reply:
ah, right. of course.